idea-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -25,11 +25,12 @@
     </project-components>
 
     <extensions defaultExtensionNs="com.intellij">
+        <highlightingPassFactory implementation="arrow.meta.ide.ArrowHighlightingPassFactory"/>
         <applicationInitializedListener implementation="arrow.meta.ide.MetaRegistrar"/>
     </extensions>
 
     <extensions defaultExtensionNs="org.jetbrains.kotlin">
-        <syntheticResolveExtension implementation="arrow.meta.ide.phases.resolve.MetaSyntheticResolveExtension" />
+        <!-- syntheticResolveExtension implementation="arrow.meta.ide.phases.resolve.MetaSyntheticResolveExtension" /-->
```

Comment:
`SyntheticResolveExtension` and `PackageFragmentProvider` come from the compiler API. We could move cli specific parts of the quote system to the compiler to prevent confusion, but for now, we should keep them registered for consistency reasons. 
```suggestion
        <syntheticResolveExtension implementation="arrow.meta.ide.phases.resolve.MetaSyntheticResolveExtension" />
```

Reply:
Hm, I don't fully understand the note about the compiler API. Isn't this just enabled for the IDE plugin via plugin.xml? 
I couldn't find out what it does for the IDE and the resolving, at least I couldn't find a feature which was broken after I disabled this.
If you want me to, I could reenable this, of course.

Reply:
Yes, please remove your change here. 
We should investigate this behavior in another dedicated PR.

Reply:
Okay. Fixed.